rapids/test_rapids.py

```diff
@@ -28,36 +29,32 @@ def _run_dask_test_script(self, name):
         self.assert_instance_command(name, verify_cmd)
 
     def verify_spark_instance(self, name):
-        self.assert_instance_command(name, "nvidia-smi")
+        self.assert_dataproc_job(
+            self.name, "pyspark", "{}/{}".format(self.INIT_ACTIONS_REPO,
+                                                 self.SPARK_TEST_SCRIPT_FILE_NAME))
```

Comment:
This is slighty off, the correct directory is as follows:
```
            self.name, "pyspark", "{}/rapids/{}".format(self.INIT_ACTIONS_REPO,
                                                 self.SPARK_TEST_SCRIPT_FILE_NAME))
```

Reply:
Thanks, Brad. Fixed it